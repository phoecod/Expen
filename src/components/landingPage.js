import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const landingPage = () => (
	<div>
		<ExpenseListFilters />
		<ExpenseList />
	</div>
);

export default landingPage;