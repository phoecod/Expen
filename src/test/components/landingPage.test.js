import React from 'react';
import {shallow} from 'enzyme';
import landingPage from '../../components/landingPage';

test('should render expense landing page', () => {
	const wrapper = shallow(<landingPage />);
	expect(wrapper).toMatchSnapshot();
});