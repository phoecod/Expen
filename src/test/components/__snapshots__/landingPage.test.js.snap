// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense landing page 1`] = `<landingPage />`;
